relay/polkadot/src/lib.rs

```diff
@@ -2566,7 +2610,63 @@ mod remote_tests {
 			})
 			.build()
 			.await
-			.unwrap();
+			.unwrap()
+	}
+
+	#[tokio::test]
+	async fn dispatch_all_proposals() {
+		if var("RUN_OPENGOV_TEST").is_err() {
+			return
+		}
+
+		sp_tracing::try_init_simple();
+		let mut ext = remote_ext_test_setup().await;
+		ext.execute_with(|| {
+			type Ref = pallet_referenda::ReferendumInfoOf<Runtime, ()>;
+			type RefStatus = pallet_referenda::ReferendumStatusOf<Runtime, ()>;
+			use sp_runtime::traits::Dispatchable;
+			let all_refs: Vec<(u32, RefStatus)> =
+				pallet_referenda::ReferendumInfoFor::<Runtime>::iter()
+					.filter_map(|(idx, reff): (_, Ref)| {
+						if let Ref::Ongoing(ref_status) = reff {
+							Some((idx, ref_status))
+						} else {
+							None
```

Comment:
We can eventually clean all the old Referenda, or? Currently they just seem to bloat storage.